app/router.js

```diff
@@ -34,9 +34,8 @@ Router.map(function() {
   this.route('blog', {path: 'blogs/:slug'}, subpageRoutes);
   this.route('stream', {path: 'streams'});
   this.route('playlist', {path: 'streams/:slug'});
-  this.route('schedule', function() {
-    this.route('date', {path: ':year/:month/:day'});
-  });
+  this.route('schedule-redirect', { path: 'schedule' });
+  this.route('schedule', { path: 'schedule/:year/:month/:day' });
```

Comment:
I can rearchitect back to what it was, if that makes more sense to you, which was `schedule` -> `date`.

The trade-off with this is that there's one extra file, `app/routes/schedule-redirect.js` but the benefit is that the routes are more descriptive of what they do, instead of `schedule` (which would redirect) and `date` (which would show the schedule).